gpbft/gpbft.go

```diff
@@ -645,49 +645,55 @@ func (i *instance) beginCommit() {
 }
 
 func (i *instance) tryCommit(round uint64) error {
-	// Unlike all other phases, the COMMIT phase stays open to new messages even after an initial quorum is reached,
-	// and the algorithm moves on to the next round.
-	// A subsequent COMMIT message can cause the node to decide, so there is no check on the current phase.
+	// Unlike all other phases, the COMMIT phase stays open to new messages even
+	// after an initial quorum is reached, and the algorithm moves on to the next
+	// round. A subsequent COMMIT message can cause the node to decide, so there is
+	// no check on the current phase.
 	committed := i.getRound(round).committed
 	quorumValue, foundStrongQuorum := committed.FindStrongQuorumValue()
 	timedOut := atOrAfter(i.participant.host.Time(), i.phaseTimeout)
 	phaseComplete := timedOut && committed.ReceivedFromStrongQuorum()
 
-	if foundStrongQuorum && !quorumValue.IsZero() {
-		// A participant may be forced to decide a value that's not its preferred chain.
-		// The participant isn't influencing that decision against their interest, just accepting it.
+	switch {
+	case foundStrongQuorum && !quorumValue.IsZero():
```

Comment:
Just to affirm my understanding, a strong quorum of commits in whichever round, leads to decide and termination.

Reply:
...as long as it's not commit to bottom 👍